cmd/restic/global.go

```diff
@@ -382,15 +384,42 @@ func ReadPasswordTwice(gopts GlobalOptions, prompt1, prompt2 string) (string, er
 	return pw1, nil
 }
 
+func ReadRepo(opts GlobalOptions) (string, error) {
+	if opts.Repo == "" && opts.RepositoryFile == "" {
+		return "", errors.Fatal("Please specify repository location (-r or --repository-file)")
+	}
+
+	repo := opts.Repo
+	if opts.RepositoryFile != "" {
+		if repo != "" {
+			return "", errors.Fatal("-r and --repository-file are mutually exclusive options")
```

Comment:
Suggestion: Starting the sentence with "Option -r and --repository-file ..." might be nicer, even if it's just cosmetics.

Reply:
I like your suggestion. I should add that it was copied from `Password file and command are mutually exclusive options` in the same file, however.